docs/content/design/from-vcs2user.md

```diff
@@ -130,4 +130,4 @@ In _Download protocol_ it is used to check whether or not the module is already
 
 Otherwise, _Download protocol_ uses `Stasher` to download module, store it into a `storage` and then it serves it back to the user.
 
-You can also see `addons.WithPool` in a code snippet above. This addon is something similar to `Stash wrapper - Pool`. It controls the number of concurrent requests proxy can handle.
\ No newline at end of file
+You can also see `addons.WithPool` in a code snippet above. This addon is something similar to `Stash wrapper - Pool`. It controls the number of concurrent requests proxy can handle.
```

Comment:
Not sure why this is being picked up as a change?

Reply:
invisible characters 😄